packages/type-merging-directives/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@graphql-tools/type-merging-directives",
```

Comment:
Does this mean that “@computed” will also go through the extra package, then? I actually like this from the doc standpoint. The main type merging page can just focus on static config with links over to and SDL page explaining how each static config translates to SDL.